openlibrary/plugins/worksearch/subjects.py

```diff
@@ -162,6 +162,21 @@ def normalize_key(self, key):
     def process_key(self, key):
         return key
 
+class subject_history(delegate.page):
+    path = '/subjects/history'
+    encoding = 'json'
+    
+    API_ROWS_LIMIT = 100
+    @jsonapi
+    def GET(self):
+        i = web.input()
```

Comment:
this spacing is wrong, does this run?

Reply:
Could you please be slightly more specific? Where is the spacing wrong?